### ISSUE TITLE
How to change border color

### ISSUE THREAD

**Issue by husstahn** [nathanreyes/v-calendar#309]
Is there a way to change vc-border-gray-400 without css, perhaps using props or theme?

**Comment by nathanreyes** [comment]
Yes, there is a `theme` prop that you can use to override the default theme keys. Still working on the documentation for this but you can [reference the structure](https://github.com/nathanreyes/v-calendar/blob/next/src/utils/defaults/theme.js) to get an idea of the sections you can configure.

The setting you want is `container`.

```html
<v-calendar :theme="theme" />
```

```js
export default {
  data() {
    return {
      theme: {
        container: {
          light: 'light-container-class',  // Classes to apply for light mode
          dark: 'dark-container-class',  // Classes to apply for dark mode
        }
      }
    };
  }
}
```

**Comment by johanbove** [comment]
[Regarding v-calendar#309:](https://github.com/nathanreyes/v-calendar/issues/309#issuecomment-489623944)

👍

**Comment by newlifehaonan** [comment]
the theme props doesn't seem to work in `@next` version
